packages/components/inputs/rich-text-utils/src/plugins/mark.tsx

```diff
@@ -1,6 +1,40 @@
 import { isKeyHotkey } from 'is-hotkey';
 import { warning } from '@commercetools-uikit/utils';
 import memoize from 'lodash/memoize';
+import { ReactNode } from 'react';
+import { ReactJSXElement } from '@emotion/react/types/jsx-namespace';
+
+type TOptions = {
```

Comment:
Maybe a name like `TMarkPluginOptions` would be more specific and relatable to its use.

Reply:
Thanks Carlos. Will work on all the feedback